pkg/codegen/docs/gen.go

```diff
@@ -1536,6 +1537,35 @@ func sortIndexEntries(entries []indexEntry) {
 	sort.Sort(sorter)
 }
 
+// getLanguageLinks returns a map of links for the current module's language-specific
+// docs by language.
+func (mod *modContext) getLanguageLinks() map[string]string {
+	languageLinks := map[string]string{}
+	for _, lang := range supportedLanguages {
+		var link string
+		var title string
+		var langTitle string
+
+		docLangHelper := getLanguageDocHelper(lang)
+		switch lang {
+		case "csharp":
+			langTitle = "C#"
```

Comment:
Consider using ".NET" here, since one could use C#, F#, or VB (or any .NET language, really). This would align with how this change uses "Node.js" rather than "TypeScript" and "JavaScript".

```suggestion
			langTitle = ".NET"
```